packages/kit/src/exports/public.d.ts

```diff
@@ -430,6 +430,31 @@ export interface KitConfig {
 		 */
 		errorTemplate?: string;
 	};
+	/**
+	 * Image optimization configuration
+	 */
+	images?: {
+		/**
+		 * Path to a a file that contains a loader that will be used to generate the an image URL out of the given source and width.
+		 * It optionally also takes third parameter for options.
+		 *
+		 * ```js
+		 * export default function loader(src, width, opts) {
```

Comment:
can we move `width` inside the `opts`? it seems a bit surprising to me to just have that one single parameter outside the options